src/index.js

```diff
@@ -1970,17 +1974,14 @@ const initializeFormElements = () => {
       }
     } catch (err) {
       console.error(err);
-      signTypedDataV3VerifyResult.innerHTML = `Error: ${err.message}`;
+      signTypedDataVerifyResult.innerHTML = `Error: ${err.message}`;
```

Comment:
this is an unrelated fixed, but encountered the issue along the way: we were using the singTypedDataV3 field instead of the correct typed data version